test/simulator/main/org/apache/cassandra/simulator/paxos/AccordClusterSimulation.java

```diff
@@ -39,28 +38,6 @@ public AccordClusterSimulation create(long seed) throws IOException
             random.reset(seed);
             return new AccordClusterSimulation(random, seed, uniqueNum, this);
         }
-
-        public void applyHandicaps()
```

Comment:
been running in a loop locally and mostly looks fine to remove... since I was touching simulator figure fast to finally close this task...